K2Bridge.Tests.End2End/QueryControllerErrorsTests.cs

```diff
@@ -38,6 +37,7 @@ public class QueryControllerErrorsTests : KustoTestBase
         public async Task MSearch_WithInvalidQuery_ReturnsElasticErrorResponse()
         {
             var response = await K2Client().MSearch(INDEX, $"{FLIGHTSDIR}/MSearch_Invalid_Query.json", false);
+            response.SelectToken("responses[*].error.root_cause[*].reason").Parent.Remove();
```

Comment:
why did you remove it

Reply:
The `reason` error message contains the query, so every time we change the query the test fails.

Reply:
The question is, how many times is it changed, the answer is while developing.
the pros, of leaving it, is that its easier to read the code and test.
I vote keep it

Reply:
Reverted the change